### PR TITLE
[Fix][core,client] Resolve the exception when starting from the specified savepoint

### PR DESCRIPTION
**Pull request by zhongjingq**

### CODE REVIEW

File: dlink-client/dlink-client-1.13/src/main/java/com/dlink/cdc/kafka/KafkaSinkJsonBuilder.java [hunk 1]
```diff
@@ -58,7 +74,7 @@ public String getHandle() {
 
     @Override
     public SinkBuilder create(FlinkCDCConfig config) {
-        return new KafkaSinkJsonBuilder(config);
+        return new KafkaSinkBuilder(config);
     }
```
Comment (aiwenmo) [inline dlink-client/dlink-client-1.13/src/main/java/com/dlink/cdc/kafka/KafkaSinkJsonBuilder.java]: Why return new KafkaSinkBuilder?

File: dlink-client/dlink-client-1.13/src/main/java/com/dlink/cdc/starrocks/StarrocksSinkBuilder.java [hunk 2]
```diff
@@ -118,18 +171,96 @@ public void addSink(
...
             }
-            return Instant.ofEpochMilli((long) value).atZone(sinkZoneIdUTC).toEpochSecond();
+        } else {
+            return value;
         }
-        return object;
+    }
+
+    protected DataStream<RowData> buildRowData1(
+            SingleOutputStreamOperator<Map> filterOperator,
```
Comment (aiwenmo) [inline dlink-client/dlink-client-1.13/src/main/java/com/dlink/cdc/starrocks/StarrocksSinkBuilder.java]: Please rename to include meaning.

**Review by aiwenmo — CHANGES_REQUESTED** [review]
Hi. It's a good fix. And please check these  feedback.

File: dlink-client/dlink-client-1.13/src/main/java/com/dlink/cdc/kafka/KafkaSinkJsonBuilder.java (hunk 1 above)
Comment (zhongjingq) [reply dlink-client/dlink-client-1.13/src/main/java/com/dlink/cdc/kafka/KafkaSinkJsonBuilder.java]: 哦sorry，在合并拷贝没有注意

File: dlink-client/dlink-client-1.13/src/main/java/com/dlink/cdc/starrocks/StarrocksSinkBuilder.java (hunk 2 above)
Comment (zhongjingq) [reply dlink-client/dlink-client-1.13/src/main/java/com/dlink/cdc/starrocks/StarrocksSinkBuilder.java]: 已修复重命名

**Review by aiwenmo — APPROVED** [review]

**Pull request merged**